src/test/java/com/tstepnik/planner/PlannerApplicationTests.java

```diff
@@ -5,9 +5,4 @@
 
 @SpringBootTest
 class PlannerApplicationTests {
```

Comment:
Why this class is still here? It should be removed.

Reply:
https://github.com/tstepnik/planner/pull/32#discussion_r423095779

Reply:
?

Reply:
This was on the develop before last merging i probably add it incidentally before merge with feature.